frontend/src/components/content-grid/preview/generic.vue

```diff
@@ -39,5 +51,15 @@ c-anchor-root.group.w-full(class="max-w-[416px]")
         | {{ viewership }} {{ viewershipName + (viewership === 1 ? '' : 's') }}
         | •
         | {{ dateToElapsedTimeString(beginAt) }}
-    c-icon.text-slate-300(v-if="icon" :name="icon")
+    .flex.flex-col.justify-between.items-center
+      template(v-if="kebabMenu")
+        c-dropdown.flex.justify-center(class="[&_>_ul]:w-fit [&_>_ul]:right-0 [&_>_ul]:top-full")
+          c-dropdown-item(class="[&_>_li]:!bg-frost-blue hover:[&_>_li]:!bg-frost-deep")
+            router-link.block(:to="{ name: 'edit', params: { videoId: videoId } }")
+              span Edit video
+              c-anchor-cover
+
+          template(#button="{ toggled }")
+            c-button.text-xl(class="!p-0 !px-2" :theme="toggled ? 'blue' : 'invisible'") ⋮
```

Comment:
You can use padding="small" instead of overriding the classes. The y padding might not look great, but the click area is too small otherwise anyway. Also, we should avoid overriding classes as much as possible as it's terrible for maintenance - instead, stylistic choices should be passed down as props so that the component has full control of its style (otherwise we start defeating the point of having the component inn the first place).